todo_app/SeleniumTest/Tests/test_e2e.py

```diff
@@ -0,0 +1,41 @@
+from os import environ, path
+from threading import Thread
+#from todo_app.data.trello import createNewBoard, deleteBoard
+from todo_app import app
+import pytest
+from unittest.mock import patch, Mock
+from selenium import webdriver
+import os
+from todo_app.data.trello  import Trello
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    trello =Trello()
```

Comment:
This works when run in VSCode but not from a command line which we should also support; the difference being that Pytest doesn't automatically load environment variables whereas the VSCode runner does. To resolve that, we should add some similar lines to our integration test to the very top of this function, but loading in the real `.env` file, e.g.
```python
    file_path = find_dotenv('.env')
    load_dotenv(file_path, override=True)
```
(Plus the appropriate imports!)